.drone.yml

```diff
@@ -63,6 +63,23 @@ steps:
       ref:
       - refs/tags/*
 
+  - name: publish latest release docker manifest
+    image: plugins/manifest
+    settings: 
+      username:
+        from_secret: docker_username
+      password:
+        from_secret: docker_password
+      target: "dessalines/lemmy-ui:latest"
```

Comment:
Instead of target, we should be able to use the following params, to avoid copying the whole block (and possibly even building twice).

```
    repo: dessalines/lemmy-ui
    tags:
      - latest
      - "${DRONE_TAG}"
```

https://docs.drone.io/plugins/popular/docker/

Reply:
I tried that, it didn't work.

Reply:
We arent using auto_tag though as far as i can see. Anyway, at least it seems to cache the build result.